src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -9,10 +9,10 @@
   "generatorVersions": "[1.0.0.0-*)",
   "description": "A project template for creating a Blazor server app that runs server-side inside an ASP.NET Core app and handles user interactions over a SignalR connection. This template can be used for web apps with rich dynamic user interfaces (UIs).",
   "groupIdentity": "Microsoft.Web.Blazor.Server",
-  "precedence": "8000",
-  "identity": "Microsoft.Web.Blazor.Server.CSharp.6.0",
+  "precedence": "9000",
+  "identity": "Microsoft.Web.Blazor.Server.CSharp.7.0",
   "shortName": "blazorserver",
-  "thirdPartyNotices": "https://aka.ms/aspnetcore/6.0-third-party-notices",
+  "thirdPartyNotices": "https://aka.ms/aspnetcore/7.0-third-party-notices",
```

Comment:
@mkArtakMSFT could you update/create this aka.ms link?

Reply:
Just created. The link should be active soon.